chsdi/models/vector/uvek.py

```diff
@@ -148,37 +148,74 @@ class MountainbikelandSperrungenUmleitungen(Base, SperrungenUmleitungen, Vector)
 
 
 class OevHaltestellen:
+    __bodId__ = 'ch.bav.haltestellen-oev'
+
+
+class OevHaltestellenZoom1(Base, OevHaltestellen, Vector):
     __tablename__ = 'oev_haltestellen_tooltip'
     __table_args__ = ({'schema': 'bav', 'autoload': False, 'extend_existing': True})
     __template__ = 'templates/htmlpopup/oev_haltestellen.mako'
-    __bodId__ = 'ch.bav.haltestellen-oev'
+    __minscale__ = 1
+    __maxscale__ = 3000
```

Comment:
@faselm this might probably explain that the tooltips are not working. As it is configured now, tooltips are disabled in scales  > 3000. Is that what the data owner wants?

Reply:
I will remove the last two commits...